frontend/src/views/Instructors/InstructorsList/InstructorsList.tsx

```diff
@@ -36,12 +91,20 @@ export const InstructorsList = () => {
   if (instructorsError || qualificationsError) {
     return <GeneralAPIError />;
   }
-
-  if (instructorsData === undefined || qualificationsData === undefined) {
+  if (qualificationsData === undefined) {
     return <FullPageLoading />;
   }
 
-  const rows = instructorsData.instructors;
+  const qualificationsOptions: PicklistOption[] = [];
+  qualificationsData.categories.forEach((quali) => {
+    qualificationsOptions.push({ value: quali.id, label: quali.name });
+  });
+
+  const rows = instructorsData?.instructors;
+  const totalRows = instructorsData?.total ?? 0;
+  const activeColumnWidth = '140px';
+  const columnSize = '25%';
+
```

Comment:
I can't add it on a proper line (line 115) but let's set it's height to `LAYOUT_HEIGHT`